vividus-plugin-web-app/src/main/resources/properties/profile/web/headless/chrome/profile.properties

```diff
@@ -1,2 +1,2 @@
 selenium.browser=chrome
-web.driver.CHROME.command-line-arguments=headless
+web.driver.CHROME.command-line-arguments=--headless --hide-scrollbars
```

Comment:
Why it is needed? :)

Reply:
to avoid useless multiple scrollbars on full-page screenshots